book/src/paper-wallet/usage.md

```diff
@@ -0,0 +1,15 @@
+# Paper Wallet Usage
+
+Solana commands can be run without ever saving a keypair to disk on a machine. If avoiding writing a private key to disk is a security concern of yours, you've come to the right place.
+
+{% hint style="warning" %}
+Even using this secure input method, it's still possible that a private key gets written to disk by unencrypted memory swaps. It is the user's responsibility to protect against this scenario.
+{% endhint %}
+
+---
+
+TODO
```

Comment:
Paging #6474, let's not add new TODOs into the code base please

Reply:
Added this issue: https://github.com/solana-labs/solana/issues/7160